update_usage.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env python3
+
+import re, sys, os
+import subprocess
+from subprocess import Popen, PIPE
+
+with open('docs/usage.md') as f:
+  content = f.read()
+
+command = subprocess.run(["cargo", "build"])
+if command.returncode != 0:
+    print('Cargo build exited with code ' + command.returncode)
+
+build_dir = os.path.dirname(os.path.realpath(__file__))
+target_dir = os.path.join(build_dir, "target", "debug")
+
+for p in os.listdir(target_dir):
+    if p.startswith("casr-") and os.path.isfile(target := os.path.join(target_dir, p)) \
```

Comment:
Maybe replace the if condition with its opposite and add `continue` as it done below for better code readability?

Reply:
I think, nesting level is acceptable